### ISSUE TITLE
[18SJ]  undefined method player? in "Select A.W. Edelswärd's next corporation" step

### ISSUE THREAD

**Issue by HelenCheale** [tobymao/18xx#9895]
[Id:140833](https://18xx.games/game/140833)

Game not loading

**How to Reproduce the Bug?**

**Which Browser / Platform?**

**Comment by benjaminxscott** [comment]
last working action https://18xx.games/game/140833?action=37

> "undefined method `player?' for #<Engine::Game::G18SJ::Game::WithNameAdapter:0x346e @host=<Engine::Player - A.W. Edelswärd> @receivership=<Engine::Player - NuttyH>>"

**Comment by HelenCheale** [comment]
Hi any idea if or when this issue will be fixed?

**Comment by HelenCheale** [comment]
Hello do you have any idea if or when this issue will be fixed please?

**Comment by HelenCheale** [comment]
Hi again do you have any idea whether this game is fixable or not, if you
could respond please that would be great.

**Comment by benjaminxscott** [comment]
I added / updated the issue title to make it easier for someone to fix it, I haven't personally played or understand 18SJ rules to fix it

**Comment by dfannius** [comment]
I have been informed that game 140833 has been deleted per the request in #10067.

**Comment by HelenCheale** [comment]
Thank you so much for dealing with this issue.

**Comment by perwestling** [comment]
I am assigning myself to this, as a candidate for my backlog. Have not been doing any work for almost a year so no promises.
If someone else want to have a look, please go ahead.

**Comment by HelenCheale** [comment]
Hi thank you very much for the offer but the game has now been deleted.

Many thanks.

**Comment by perwestling** [comment]
Yes, but I want to see if I can fix the issue to avoid it appearing again.

**Comment by HelenCheale** [comment]
Ok well thank you very much and I wish you lots of luck.

**Comment by perwestling** [comment]
@HelenCheale Thanks for trying out 18SJ 2 player variant. I hope you do try it again, now that this issue is fixed.

The game has been deleted, and the gist above did not work, so I used the info there to recreate this issue. The bug did appear in the end of SR and to me seems to always happen.

So I made a correction, and now this bug do not occur, but instead I managed to play several SRs without any issues.

Have now submitted a correction, which will make it 18SJ 2 player variant playable again, when the correction is deployed.
